CONTRIBUTING.md

```diff
@@ -163,7 +163,71 @@ in general try to follow them.
+- Ensuring that a bug can be reproduced, and if possible, the behavior can be traced back to the offending code or documentation.
+- Determining whether a feature request belongs in a component, should be accomplished through other means, or isn't appropriate for a component at this time.
+- Guiding any interested parties to another person or resource that may be more knowledgeable about an issue.
+- Suggesting an issue for discussion at a SIG meeting if a synchronous discussion would be more productive.
+
+#### Issue assignment
+
+Issues are assigned for someone to work on by a triager when someone volunteers to work on an issue. Assignment is intended to prevent duplicate work by making it visible who is
+working on a particular task. A person who is assigned to the issue may be assigned to help triage the issue and implement it, or can be assigned after the issue has already been
+triaged and is ready for work. If someone who is assigned to an issue is no longer able to work on it, they may request to be unassigned from the issue.
+
+### Label Definitions
+
+| Label                | When to apply                                                                                                                                                                                           |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `bug`                | Something that is advertised or intended to work isn't working as expected.                                                                                                                             |
+| `enhancement`        | Something that isn't an advertised feature that would be useful to users or maintainers.                                                                                                                |
+| `good first issue`   | Implementing this issue would not require specialized or in-depth knowledge about the component and is ideal for a new or first-time contributor to take.                                               |
+| `help wanted`        | The code owners for this component do not expect to have time to work on it soon, and would welcome help from contributors.                                                                             |
+| `needs discussion`   | This issue needs more input from the maintainers or community before work can be started.                                                                                                               |
+| `needs triage`       | An issue should be considered triaged when it has been validated to be a genuine bug or enhancement that the maintainers or code owners expect will be worked on.                                       |
```

Comment:
This is where I'm fuzzy, how do we validate? Do we need an ack from someone or can we go by our own judgement?

Reply:
That's a good point. It's really by the judgement of the triager or code owner. Obviously changes can always arise that will change the status, but in general I think it should be used as a flag that the issue is ready for work. I'll try to update this description to offer more guidelines. It may also make sense to give this its own section.